src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: $square-size * 7 + $gap + $calendar-pading;
```

Comment:
You have not enough width for 7 days in 1 row. Here a screenshot how it looks. Maybe you should include the variable with the amount of gaps (you have 6 gaps) and multiple it to your gap size ($gap).

https://prnt.sc/I4M60bmsT6wW